packages/utils/src/time.ts

```diff
@@ -135,14 +135,25 @@ export const browserPerformanceTimeOrigin = ((): number | undefined => {
   if (!performance) {
     return undefined;
   }
-  if (performance.timeOrigin) {
+  // If performance APIs are over an hour skewed don't use them
+  const THRESHOLD = 3600 * 1000;
+  // eslint-disable-next-line deprecation/deprecation
+  const performanceTiming = performance.timing && performance.timing.navigationStart;
```

Comment:
If this is `0`, this might be falsey in the if conditions below. Might be good to create a variable to ensure its existence. Like `const hasPerformanceTiming = typeof performanceTiming === 'number';`

Reply:
If this is 0 we don't want to use it anyways right? Which I think will get prevented being used by
`Math.abs(navigationStart + performance.now() - Date.now()) < THRESHOLD`

Reply:
You'll want to use it. But there are conditions that will fail the short-circuit test in unexpected ways.

Reply:
Updated in cb5afad